src/api/app/models/package.rb

```diff
@@ -471,7 +471,7 @@ def update_issue_list
 
   def parse_issues_xml(query, force_state = nil)
     begin
-      answer = Suse::Backend.post(self.source_path(nil, query), nil)
+      answer = Suse::Backend.post self.source_path(nil, query)
```

Comment:
A lot of parenthesis, I thought it was more readable without them, but I can add them again if you want. :no_mouth: